src/uproot/streamers.py

```diff
@@ -672,13 +672,24 @@ def class_code(
         member_names,
         class_flags,
     ):
+        read_member_n.append(
+            """
+        forth = False
+        forth_obj = None
+        fcode_pre = []
+        fcode_post = []
+        temp = None
+        if "forth" in context.keys():
+            forth = True
+            forth_obj = context["forth"]
+            key = forth_obj.get_key()\n"""
+        )
```

Comment:
I think you don't want to put code into the `read_member_n`, which is for the memberwise format. (Unless a particular type has memberwise implemented in Python as well.)